app/reducers/routing.js

```diff
@@ -1,19 +1,18 @@
+// @flow
 import { Routing } from 'app/actions/ActionTypes';
+import produce from 'immer';
 
 const initialState = {
   statusCode: null
 };
 
-export default function routing(state = initialState, action) {
+type State = typeof initialState;
```

Comment:
:biohazard:

Reply:
This is used a few other places, is there something wrong with it?

https://github.com/webkom/lego-webapp/blob/60aae52f29b89f1f7bd23150d952df69b38ac090/app/reducers/meetingsToken.js#L12
https://github.com/webkom/lego-webapp/blob/60aae52f29b89f1f7bd23150d952df69b38ac090/app/reducers/notificationsFeed.js#L11
https://github.com/webkom/lego-webapp/blob/963d4dd22231e59c796661ed61076b89687c7c5a/app/reducers/search.js#L119

Reply:
Yee, typing is kinda bad. I guess it works, but it isn't that expressive. And when `statusCode` is `null`, what is then the type? :man_shrugging:

Reply:
Maybe todo for typescript? I can add better types here, but it will have to be changed soon anyways, and I don't know if we want to do too much in one PR.